### ISSUE TITLE
App crash adding new contact

### ISSUE THREAD

**Issue by carlbennetts** [status-im/status-react#649]
"When copy pasting too many characters into the box, and then trying to scroll, the app crashes."

https://instabug.com/applications/status-10bb8cbc-15f6-44f0-8c7a-db478dc34d7d/beta/bugs/94
https://instabug.com/applications/status-10bb8cbc-15f6-44f0-8c7a-db478dc34d7d/beta/bugs/131

**Comment by rasom** [comment]
I cannot reproduce this in `develop`.
Does anyone have clear steps to reproduce? I'm trying to past "doublesized" public key, not sure what exactly means  "too many characters".

**Comment by jumarko** [comment]
@rasom, I copied your comment
> I cannot reproduce this in develop.
Does anyone have clear steps to reproduce? I'm trying to past "doublesized" public key, not sure what exactly means "too many characters".

to the public key field and I ended up with following error:
![screen](https://cloud.githubusercontent.com/assets/1083629/22433792/dd4b9a08-e719-11e6-837e-327f50071f2e.png)

This is from android log:
```
01-30 12:29:19.821  7057  7097 E ReactNativeJS: Does anyone have clear steps to reproduce? I'm trying to past \"doublesized\" public key, not sure what exactly means \"too many chracters\".":1:53: Invalid characters in string constant.
```
It might be that new lines in that string are problematic.
However, This probably wasn't the original problem discussed in this bug O:-).
